src/routes/home/Home.js

```diff
@@ -1393,23 +1394,32 @@ class Home extends React.Component {
               }}
             >
               <summary>
-                Previous Submissions
-                {this.state.submissions.length > 0 &&
-                  ` (${this.state.submissions.length})`}
+                Previous Submissions (
+                {this.state.submissions.length > 0
+                  ? this.state.submissions.length
+                  : 'loading...'}
+                )
               </summary>
 
-              <ul>
-                {this.state.submissions.length === 0
-                  ? 'Loading submissions...'
-                  : this.state.submissions.map(submission => (
+              {this.state.submissions.length === 0 ? (
+                'Loading submissions...'
+              ) : (
+                <>
+                  <CSVLink data={this.state.submissions}>
+                    Download as CSV
+                  </CSVLink>
```

Comment:
An outstanding issue here is that the objects in the `submissions` array contain properties whose values are themselves objects, so they show up as `[object Object]` in the CSV that `react-csv` generates. I tried doing `JSON.stringify` on the nested objects, but that didn't work because `react-csv` appears not to escape commas properly.

Some options are:

* escape the commas correctly using another library?
* use semicolons, tabs, or pipes instead of commas as separators, see https://github.com/react-csv/react-csv/issues/3
* do nothing, just let `[object Object]` stay in there

Reply:
I fixed this by using tabs after doing the `JSON.stringify`, see https://github.com/josephfrazier/reported-web/pull/413/files/12621774cee15c8714ebf08f6ae135ae88fa043a..3636c5346fff5ffc985cd5719aad7648bebf470b